lib/collection/src/update_handler.rs

```diff
@@ -433,18 +433,18 @@ impl UpdateHandler {
     /// Checks conditions for all optimizers and returns whether any is satisfied
     ///
     /// In other words, if this returns true we have pending optimizations.
-    pub(crate) fn has_non_optimal_segments(&self) -> bool {
-        // If we did trigger optimizers at least once, we do not consider to be pending
-        if self.has_triggered_optimizers.load(Ordering::Relaxed) {
-            return false;
-        }
+    pub(crate) fn check_suboptimal_optimizations(&self) -> (bool, bool) {
```

Comment:
I find `check_suboptimal_optimizations` rather confusing and would prefer to call it something like `check_optimizer_conditions`. Wdyt?

Reply:
Sounds better. thanks!